### ISSUE TITLE
Calculate Soli only for `sn` level

### ISSUE THREAD

**Issue by mjbloemer** [iza-institute-of-labor-economics/gettsim#445]
Currently, soli_st.py calculates the Soli for a variable `eink_st_per_individual`:

https://github.com/iza-institute-of-labor-economics/gettsim/blob/4379a90b5639e64628a5cb14cd411b9225b12240/gettsim/taxes/soli_st.py#L38

 This is not what is done in reality and might be prone to errors or misinterpretations. I think the concept "per individual" is also not needed.

Soli should be calculated for `eink_st_mit_kinderfreib_tu`.

There are two parameters for Solifreigrenzen (https://www.buzer.de/gesetz/282/a3297.htm): Currently 16956 Euro and 33912 for joint taxation. The second parameter is simply twice the first one and could be calculated by gettsim.

**Comment by hmgaudecker** [comment]
But what you describe in the title is exactly what is happening, that is why the function is called `soli_st_tu`.

The above is an implementation detail within that functon -- sure we could be closer to the law, but is there any mathematical issue with it?

**Comment by ChristianZimpelmann** [comment]
> The above is an implementation detail within that functon -- sure we could be closer to the law, but is there any mathematical issue with it?

I agree and this is how I understood @mjbloemer's point. We shouldn't change the name of the function, but the implementation can be improved.

Don't think there is a mathematical issue.

**Comment by hmgaudecker** [comment]
For the moment, I would suggest sticking to documenting the internals better to avoid confusion?

**Comment by MImmesberger** [comment]
Yes!

**Comment by hmgaudecker** [comment]
Done.
